appMemoryScreen.js

```diff
@@ -0,0 +1,165 @@
+
+const memory = document.querySelector('#memory-screen')
+const gameScreen = document.querySelector('#game-screen')
+const testScreen = document.querySelector('#test-screen')
+const inputBox = document.querySelector('#input-box')
+const startClick = document.querySelector('#start-button')
+const cardScreen = document.querySelector('#card-screen')
+const playClick = document.querySelector('#play-button')
+const tests = document.querySelector('#test')
+const languageBox = document.querySelector('#language')
+
+const url = 'http://localhost:8080/play'
+
+// Axios first round
+// axios
+// .get(url)
+// .then(res => {
+    // memoryScreen(res.data)
+// })
+
+//  Dummy Data Solution
+let data = [
+    {
+        image_url: 'https://images.unsplash.com/photo-1503066211613-c17ebc9daef0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80',
+        foreignLang: 'nick',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1517849845537-4d257902454a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=975&q=80',
+        foreignLang: 'batia',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1503066211613-c17ebc9daef0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80',
+        foreignLang: 'pouyan',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1503066211613-c17ebc9daef0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80',
+        foreignLang: 'philip',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1503066211613-c17ebc9daef0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80',
+        foreignLang: 'foreign',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1495594059084-33752639b9c3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80',
+        foreignLang: 'foreign',
+        english: 'eng'
+    },
+    {
+        image_url: 'https://images.unsplash.com/photo-1503431128871-cd250803fa41?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80',
+        foreignLang: 'foreign',
+        english: 'eng'
+    }
+]
+
+// play page
+let playScreen = function(data) {
```

Comment:
I think functions should be "const" instead of "let", since they are defined once